packages/devtools_app/lib/src/screens/logging/logging_controller.dart

```diff
@@ -740,7 +741,10 @@ class LogData with SearchableDataMixin {
     this.isError = false,
     this.detailsComputer,
     this.node,
-  });
+  }) {
+    // Fetch details immediately on creation.
+    unawaited(compute());
```

Comment:
Is it possible `compute()` might throw? If so, maybe we should call `catchError`

Reply:
Good idea! done.